cert_manager/datadog_checks/cert_manager/cert_manager.py

```diff
@@ -0,0 +1,37 @@
+# (C) Datadog, Inc. 2019-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import OpenMetricsBaseCheck
```

Comment:
I view the migration already as a massive change/starting fresh so can you please use `OpenMetricsBaseCheckV2`?

Reply:
I added a new commit with this